hallux/targets/github_proposal.py

```diff
@@ -118,13 +117,37 @@ def commit_diff(self) -> bool:
                 body=body,
                 commit=commit,
                 side="RIGHT",
-                path=str(compacted.filename),
-                line=compacted.end_line,
-                start_line=compacted.start_line,
+                path=str(self.existing_proposal.filename),
+                line=self.existing_proposal.end_line,
+                start_line=self.existing_proposal.start_line
+                if self.existing_proposal.start_line > self.existing_proposal.end_line
+                else GithubObject.NotSet,
             )
-        except GithubException:
+        except BaseException as ex:
+            logger.debug(ex)
             success = False
 
+        # compacted = self.compact_proposal(self.existing_proposal)
```

Comment:
Remove this commented out code.
```suggestion
        FilesystemTarget.revert_diff(self)

```